react/finger-exercises/ecmascript/01 - spread/index.js

```diff
@@ -0,0 +1,40 @@
+import { isArray } from './utils';
+
+export function min(...numbers) {
+  if (numbers.length === 0) {
+    return undefined;
```

Comment:
functions ins javascript return `undefined` if you don't explicitly return anything. Try to re define the logic so that you don't have to return `undefined` like this

Reply:
I've changed the logic of the function to this one:
```
export function min(...numbers) {
  if (numbers.length > 1) {
    return Math.min(...numbers);
  }

  if (numbers.length === 1) {
    if (isArray(numbers[0])) {
      return Math.min(...numbers[0]);
    }

    return numbers[0];
  }
}
`
``
In this way there is no `return undefined` and my test continue passing.
But now I have this situation with the rule: consistent-return rule of Es Lint: https://eslint.org/docs/rules/consistent-return

How I can return an undefined complying with this rule?

Reply:
well you have 2 options... override the linter (forcing a commit so that the linter ignores this particular rule) or find another way to do it :D